firebase_admin/tenant_mgt.py

```diff
@@ -141,6 +194,56 @@ def get_tenant(self, tenant_id):
         else:
             return Tenant(body)
 
+    def create_tenant(
+            self, display_name=None, allow_password_sign_up=None, enable_email_link_sign_in=None):
+        """Creates a new tenant from the given parameters."""
+        payload = {}
+        if display_name is not None:
+            payload['displayName'] = _auth_utils.validate_string(display_name, 'displayName')
+        if allow_password_sign_up is not None:
+            payload['allowPasswordSignup'] = _auth_utils.validate_boolean(
+                allow_password_sign_up, 'allowPasswordSignup')
+        if enable_email_link_sign_in is not None:
+            payload['enableEmailLinkSignin'] = _auth_utils.validate_boolean(
+                enable_email_link_sign_in, 'enableEmailLinkSignin')
+
+        try:
+            body = self.client.body('post', '/tenants', json=payload)
+        except requests.exceptions.RequestException as error:
+            raise _auth_utils.handle_auth_backend_error(error)
+        else:
+            return Tenant(body)
+
+    def update_tenant(
+            self, tenant_id, display_name=None, allow_password_sign_up=None,
+            enable_email_link_sign_in=None):
+        """Updates the specified tenant with the given parameters."""
+        if not isinstance(tenant_id, str) or not tenant_id:
```

Comment:
I think `isinstance(tenant_id, str)` covers `not tenant_id` cases, right?
Also, did we choose not to use `_auth_utils.validate_string` here to provide a more specific error message?

Reply:
That doesn't cover the case where `tenant_id == ''`.

It's the same reason why we can't use `validate_string()` here.